packages/adapter-posthog/src/index.ts

```diff
@@ -0,0 +1,133 @@
+import { PostHog } from 'posthog-node';
```

Comment:
Is this Edge Runtime compatible?

Reply:
![image](https://github.com/user-attachments/assets/93933274-338d-4dd3-9987-f124e9832da6)

> Note: We use the API because [Vercel edge middleware](https://vercel.com/docs/concepts/functions/edge-middleware) (which optimizes the speed of this request in your app) has a limited number of packages and posthog-node isn’t one of them.

https://posthog.com/tutorials/nextjs-bootstrap-flags

Docs say that it is not. That was in 2023 so it's possibly out of date.